src/main/kotlin/pt/up/fe/ni/website/backend/controller/AccountController.kt

```diff
@@ -19,5 +20,5 @@ class AccountController(private val service: AccountService) {
     fun getAccountById(@PathVariable id: Long) = service.getAccountById(id)
 
     @PostMapping("/new")
-    fun createAccount(@RequestBody dto: AccountDto) = service.createAccount(dto)
+    fun createAccount(@RequestPart account: AccountDto, @RequestPart image: MultipartFile?) = service.createAccount(account)
```

Comment:
I'm having trouble getting everything together in one single DTO.
In order to support files, we need to use multipart/form-data, and that doesn't work with `@RequestBody`. We would have to use `@ModelAttribute`. However, this annotation seems to not parse the dto correctly and parse some fields incorrectly:

![image](https://user-images.githubusercontent.com/23351316/208517135-2bb0c4a7-defe-4046-920b-2498e8a586ac.png)

The only solution I found was to separate the account dto and the image file upload in two separate form fields, causing the request to look like this:

![image](https://user-images.githubusercontent.com/23351316/208517377-bc687b11-8290-4195-a2e1-4a1942e07c86.png)


Some references

https://stackoverflow.com/questions/23533355/spring-controller-requestbody-with-file-upload-is-it-possible#answer-31753785

https://stackoverflow.com/questions/61343477/how-to-send-multipart-and-requestbody-together-in-spring-boot-2-x